lib/libfrr.h

```diff
@@ -174,4 +174,36 @@ extern bool debug_memstats_at_exit;
 }
 #endif
 
+static inline time_t frr_get_realtime(struct timeval *tvo)
+{
+	if (tvo) {
+		gettimeofday(tvo, NULL);
+		return tvo->tv_sec;
+	}
+
+	return -1;
+}
+
+static inline ssize_t frr_realtime_to_string(struct timeval *tv, char *buf,
```

Comment:
time_to_string? or frrtime_to_interval?

Reply:
In any event this is not the right place to put these changes

Reply:
These are simple short utility functions just like the monotime_xxx functions in lib/monotime.h. By the way none of the current functions displays milli/microseconds granularity. Hence the need of this function.

Reply:
again we've already created all these functions.  We do not need them again and they do not belong in libfrr.h

Reply:
@donaldsharp Can you please point me to a similar function that prints timestamps with microseconds granularity? I will be very much happy to use them instead of writing new ones. I could find one and hence coded these ones. Please note frrtime_to_interval() does not print timestamps. It prints intervals in Days/Hours/Minutes/Seconds. Does not print a timestamp like "2021-01-22 07:20:39.522846".